pkgs/development/python-modules/keras/default.nix

```diff
@@ -1,55 +1,64 @@
 {
   lib,
   buildPythonPackage,
-  fetchPypi,
-  pytest,
-  pytest-cov,
-  pytest-xdist,
```

Comment:
Keras test are currently broken. I am working on bringing them back in another PR.

Reply:
Do we know if keras is actually functional after the present PR?

Reply:
> Keras tests are currently broken. I am working on bringing them back in another PR.

Not all features work, because of how old `tensorflow` is on nixpkgs (causing the `mhcflurry` and `trfl` to fail, but this was already the case).
However, the other ~100-ish downstream dependencies seem to still build fine.

Reply:
Right, tensorflow... that is an unfortunate situation

Reply:
Tracking in https://github.com/NixOS/nixpkgs/issues/316759